src/hooks/usePrevious/__stories__/usePrevious.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import usePrevious from "../index";
+import UsePrevious from "./hooksDummyComponents/UsePrevious";
+import Button from "components/Button/Button";
+import { Counter } from "components";
+import Flex from "components/Flex/Flex";
+import "./usePrevious.stories.scss";
+
+<Meta title="Hooks/usePrevious" component={UsePrevious} />
+
+# usePrevious
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+Hook for keeping previous state value.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const [count, setCount] = useState(1);
+      const prevCount = usePrevious(count);
+      const incrementButtonOnClick = () => {
+        setCount(prevCount => prevCount + 1);
+      };
+      return (
+        <div>
+          <Flex className="use-previous-counter-container" direction={Flex.directions.COLUMN}>
+            <div className="use-previous-counter-label">Current</div>
+            <Counter count={count} ariaLabeledBy="currentCounterLabel" />
+          </Flex>
+          <Flex className="use-previous-counter-container" direction={Flex.directions.COLUMN}>
+            <div className="use-previous-counter-label">Previous</div>
+            <Counter count={prevCount} ariaLabeledBy="previousCounterLabel" />
+          </Flex>
+          <Button onClick={incrementButtonOnClick}>Increment</Button>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Arguments
+
+<ArgsTable of={UsePrevious} />
```

Comment:
please see other hooks stories for consistency of hooks tables (`useClickOutside.stories.mdx` for example)

Reply:
We've been discussing it with @hadasfa. The storybook for hooks is not very consistent right now, so we've decided that we'll make it consistent and that it'll look like this: title - arguments (cause hook has arguments, not props), content - ArgsTable (cause it's more readable than FunctionArguments). What do you think?

Reply:
as far as I know we still using the props table for arguments with the "fake component" for the props api
and using the args table for the return value. @orrgottlieb 
I also think it is more readable.
Am I wrong?

Reply:
Removed dummy component and replaced `ArgsTable` with `FunctionArguments`